addons/mail/static/src/models/thread/thread.js

```diff
@@ -452,6 +469,22 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * Performs the `execute_command` RPC on `mail.channel`.
+         *
+         * @static
+         * @param {Object} param0
+         * @param {integer[]} param0.ids list of id of channels
+         * @param {string} param0.command the commadn to execute
+         */
+        static async performRpcExecuteCommand({ ids, command }) {
+            return this.env.services.rpc({
+                model: 'mail.channel',
+                method: 'execute_command',
+                args: [ids, command]
+            }, { shadow: true });
```

Comment:
Comment is usually coming from user action so I would except it to be not shadow.

Reply:
It was already shadow before so I just translated it. Don't know if we need to change that ?

Reply:
It's not very important but I would change it.

Reply:
alright !